graylog2-server/src/main/java/org/graylog2/indexer/migration/RemoteReindexMigration.java

```diff
@@ -99,4 +99,20 @@ public void finish() {
         }
         status(Status.FINISHED);
     }
+
+    /**
+     * @return How much of the migration is done, in percent, int value between 0 a 100.
+     */
+    @JsonProperty("progress")
+    public int progress() {
+        final int countOfIndices = indices.size();
+
+        if(indices.isEmpty()) {
+            return 0; // avoid division by zero
```

Comment:
I think this should return 100. If the user selects no indices to migrate, migration is done right away.

Reply:
good point!